src/services/publishing/publishing.ts

```diff
@@ -42,19 +42,14 @@ export const publishVerifiableDocumentJob = async (
   return tx.transactionHash;
 };
 
-//TODO: Move this to publish.tsx file.
 export const publishDnsDidVerifiableDocumentJob = async (
-  job: PublishingJob,
+  wrappedDocuments: any[],
```

Comment:
what happens when we type it properly? `v2.OpenAttestationDocument | v3.OpenAttestationDocument []`